specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -567,6 +853,110 @@
         }
       },
       "description": "The information of an AvailablePrivateEndpointType."
+    },
+    "PrivateDnsZoneGroup": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrivateDnsZoneGroupPropertiesFormat",
+          "description": "Properties of the private dns zone group."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Private dns zone group resource."
+    },
+    "PrivateDnsZoneGroupPropertiesFormat": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the private dns zone group resource."
+        },
+        "privateDnsZoneConfigs": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateDnsZoneConfig"
+          },
+          "description": "A collection of private dns zone configurations of the private dns zone group."
+        }
+      },
+      "description": "Properties of the private dns zone group."
+    },
+    "PrivateDnsZoneConfig": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PrivateDnsZonePropertiesFormat",
+          "description": "Properties of the private dns zone configuration."
+        }
+      },
+      "description": "PrivateDnsZoneConfig resource."
+    },
+    "PrivateDnsZonePropertiesFormat": {
+      "properties": {
+        "privateDnsZoneId": {
+          "type": "string",
+          "description": "The resource id of the private dns zone."
+        },
+        "recordSets": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RecordSet"
```

Comment:
Is this read-only? Based on the example, looks like user cannot set them.

Reply:
I think so, let me double check with Ajinkya